k6/createPosts.js

```diff
@@ -0,0 +1,63 @@
+import http from 'k6/http';
+
+const config = JSON.parse(open('../config/config.json'));
+const response = JSON.parse(open('../temp_store.json'));
+
+export const options = {
+    vus: config.PostsConfiguration.Count,
+    duration: config.PostsConfiguration.Duration,
+}
+
+function getRandomMessage(sentenceLength, wordsLength) {
+	let message = '';
+    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+	let words = 0;
+    sentenceLength = Math.floor(Math.random() * sentenceLength) + 1;
```

Comment:
I think this is the number of words in a sentence, but it sounds like its the number of letters in the sentence. Please change